docs/docsite/rst/community/collection_contributors/collection_integration_add.rst

```diff
@@ -213,7 +213,7 @@ That is enough for our very basic example.
 .. code-block:: yaml
 
   - name: Test postgresql_info module
-    become: yes
+    become: true
     become_user: postgres
     postgresql_info:
```

Comment:
Can you make the same changes to this file as well? to change `postgresl_info` to `community.postregsql.postgresql_user`?

Basically, we have to prepend all occurrences of `postgresql_` with `community.postgresql.postresql_` on all these files.  The shorter name is an old convention that is outdated.

Reply:
@samccann  just making sure you want me to change `postgresql_info`  to `community.postregsql.postgresql_user` not 
`community.postregsql.postgresql_info` right ?

Reply:
`community.postgresql.postgresql_info`